.github/workflows/install.yml

```diff
@@ -32,6 +32,7 @@ jobs:
         run: |
           pip install torch-scatter -f https://data.pyg.org/whl/torch-${{ matrix.torch-version }}+cpu.html
           pip install torch-sparse -f https://data.pyg.org/whl/torch-${{ matrix.torch-version }}+cpu.html
+          pip install torch-cluster -f https://data.pyg.org/whl/torch-${{ matrix.torch-version }}+cpu.html
```

Comment:
This seemed to be required now that we're importing cluster here - let me know if there is a good reason not to include cluster in this CI step

Reply:
IMO, you can `import torch-cluster` inside the function/transform. I would like to keep it optional.

Reply:
okay fair enough - will revert